### PR TITLE
[ADD] Wizard to change the timesheet period

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**
To avoid to delete and recreate the timesheet (loosing all timesheet lines) only if the period was wrong

### CODE REVIEW

**Comment by sbidoul** [comment]
@oca-clabot,

The contributions of @Olivier-LAURENT are covered by the ACSONE ECLA.

**Comment by gurneyalex** [comment]
@sbidoul @Olivier-LAURENT I've updated the OCA database. Next PR should not be flagged by the clabot.

**Comment by anthony-muschang** [comment]
:+1:

File: hr_timesheet_sheet_change_period/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# along with hr_timesheet_sheet_change_period.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'HR Timesheet Change Period',
+    'summary': """
+Allows to change covered period while the timesheet is in the 'draft' state""",
+    'author': 'ACSONE SA/NV',
```
Comment (gurneyalex) [inline hr_timesheet_sheet_change_period/__openerp__.py]: could you please add "Odoo Community Association (OCA)" to the list of authors? 

This is for referencing on apps.odoo.com.

**Comment by gurneyalex** [comment]
:+1: once remark above is taken into account.

**Comment by mdietrichc2c** [comment]
Travis is now green, LGTM :+1:

**Comment by charbeljc** [comment]
:+1:

**Pull request merged**